app/models/application.rb

```diff
@@ -161,6 +161,14 @@ def find_all_nearest_or_recent
     end
   end
 
+  def official_submission_period_expired?
+    if on_notice_to && Date.today > on_notice_to
+      true
+    else
+      false
+    end
```

Comment:
nil will evaluate as false so does that really matter for what you're using it for?